catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/MetacardApplication.java

```diff
@@ -347,12 +347,18 @@ public void init() {
                     .map(Result::getMetacard)
                     .map(metacard -> {
                         boolean isSubscribed = ids.contains(metacard.getId());
-
-                        return ImmutableMap.builder()
-                                .putAll(transformer.transform(metacard))
-                                .put("subscribed", isSubscribed)
-                                .build();
+                        try {
+                            return ImmutableMap.builder()
+                                    .putAll(transformer.transform(metacard))
+                                    .put("subscribed", isSubscribed)
+                                    .build();
+                        } catch (RuntimeException e) {
+                            LOGGER.warn(
```

Comment:
Does this really need to be a warn message? At warn, we should be instructing an administrator how to fix this issue. If we can't instruct someone to do something about this message, we should lower the level.